README.md

```diff
@@ -155,6 +155,38 @@ var browser = require("webextension-polyfill/dist/browser-polyfill.min");
 ...
 ```
 
+### Preferred Usage with webpack
+
+The previous section explains how to bundle `webextension-polyfill` in each script. This other method will instead include a single copy of the polyfill across your extension. You will need a special plugin:
```

Comment:
`This other method will instead include a single copy of the polyfill across your extension.` -> `An alternative method is to include a single copy of the library in your extension, and load the library as shown in [Basic Setup](#basic-setup).`

`a special plugin` -> `to install [copy-webpack-plugin](https://www.npmjs.com/package/copy-webpack-plugin)`